doc/objectives.md

```diff
@@ -0,0 +1,6 @@
+# Study objectives
+
+Are swallow populations declining? 
```

Comment:
If you want this as a bullet-point list, use dashes or asterisks at the beginning of each line, like this:

```
* Are swallow populations declining?
* Are they experiencing range shifts?
```